### ISSUE TITLE
"sendAllMetadata" not rendering "single value format" when deploying with 1.4.5 chart

### ISSUE THREAD

**Issue by matthewmodestino** [splunk/splunk-connect-for-kubernetes#554]
**What happened**:

deploying 1.4.5 logging chart and I am unable to get the `sendAllMetadata: false` to render in the logging configmap, and as such my event format has changed in Splunk across minor chart release. 

**What you expected to happen**:

The container logging format not to change in a minor release. Expected this section to render:

https://github.com/splunk/splunk-connect-for-kubernetes/blob/1989ac0c5d56df3fb849772041a0378e7e873032/helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/configMap.yaml#L423-L430

**How to reproduce it (as minimally and precisely as possible)**:

```

helm repo add splunk https://splunk.github.io/splunk-connect-for-kubernetes/

kubectl create ns canlab

helm -n splunk install sck -f sck-canlab-145.yaml splunk/splunk-connect-for-kubernetes

```

**Anything else we need to know?**:

Here is my `values.yaml`

```
#global settings
global:
  logLevel: info 
  splunk:
    hec:
      protocol: https
      insecureSSL: true 
      host: myHost
      token: "myToken"
      port: 8088 
  kubernetes:
    clusterName: "mattymo-microk8s"

#local config for logging chart
splunk-kubernetes-logging:
  #logLevel: debug
  containers:
    logFormatType: cri
    logFormat: "%Y-%m-%dT%H:%M:%S.%N%:z"
  journalLogPath: /var/log/journal
  buffer:
    "@type": memory
    total_limit_size: 600m
    chunk_limit_size: 10m
    chunk_limit_records: 100000
    flush_interval: 2s
    flush_thread_count: 1
    overflow_action: block
    retry_max_times: 10
    retry_type: periodic
  k8sMetadata:
  # Pod labels to collect
    podLabels:
      - app
      - k8s-app
      - release
      - environment
      - tier
  splunk:
    hec:
      indexName: k8s
  logs:
    sck:
      from:
        pod: sck-splunk-kubernetes-
        container: splunk-fluentd-k8s-
      multiline:
        firstline: /^\d{4}-\d{2}-\d{2}\s\d{2}\:\d{2}\:\d{2}\s\+\d{4}\s\[\w+\]\:/
        separator: "\n"
        flushInterval: 2
    
#local config for objects chart
splunk-kubernetes-objects:
  rbac:
    create: true
  serviceAccount:
    create: true
    name: splunk-kubernetes-objects
  kubernetes:
    insecureSSL: true
  objects:
    core:
      v1: [{"name":"pods", "interval":"60s"},{"name":"nodes", "interval":"60s"},{"name":"component_statuses", "interval":"60s"},{"name":"config_maps", "interval":"60s"},{"name":"namespaces", "interval":"60s"},{"name":"persistent_volumes", "interval":"60s"},{"name":"persistent_volume_claims", "interval":"60s"},{"name":"resource_quotas", "interval":"60s"},{"name":"services", "interval":"60s"},{"name":"service_accounts", "interval":"60s"},{"name":"events", "mode":"watch"},]
    apps:
      v1:
        [{"name":"daemon_sets", "interval":"60s"},{"name":"deployments", "interval":"60s"},{"name":"replica_sets", "interval":"60s"},{"name":"stateful_sets", "interval":"60s"},]
  splunk:
    hec:
      indexName: em_meta

#local config for metrics chart
splunk-kubernetes-metrics:
  metricsInterval: 60s
  kubernetes:
    kubeletPort: 10255
    kubeletPortAggregator: 10250
    useRestClientSSL: false
    insecureSSL: true
  rbac:
    create: true
  serviceAccount:
    create: true
    name: splunk-kubernetes-metrics
  splunk:
    hec:
      indexName: em_k8s

#App for Infra Dimensions for K8s
  customFilters:
    node:
      tag: "kube.node.**"
      type: record_modifier
      body: |-
        <record>
          entity_type k8s_node
        </record>
    pod:
      tag: "kube.pod.**"
      type: record_modifier
      body: |-
        <record>
          entity_type k8s_pod
        </record>
```

**Environment**: Single Ubuntu 18.04LTS Running MicroK8s
- Kubernetes version (use `kubectl version`):

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.17", GitCommit:"f3abc15296f3a3f54e4ee42e830c61047b13895f", GitTreeState:"clean", BuildDate:"2021-01-13T13:13:00Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

- Ruby version (use `ruby --version`):
- OS (e.g: `cat /etc/os-release`):

```
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

- Splunk version: 8.1.2
- Splunk Connect for Kubernetes helm chart version: 1.4.5
- Others:

```
helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

![image](https://user-images.githubusercontent.com/31549209/107854309-101bd880-6de9-11eb-8a19-b4ac6c44c4fa.png)

**Comment by matthewmodestino** [comment]
Above photo depicts log format after rollback using this handy command this taught me:

`helm -n splunk install sck -f sck-canlab-145.yaml splunk/splunk-connect-for-kubernetes --version 1.4.4`

**Comment by rockb1017** [comment]
Thanks Matt for reporting this. this is true. chart in helm repo is not properly updated. it is actually applied in opposite logic. Will fix this and release 1.4.6

**Comment by matthewmodestino** [comment]
ah! so workaround for me is to set to true?

**Comment by rockb1017** [comment]
yea. it is fixed in 1.4.6 which just got released.
